### DISCUSSION TITLE
Can't use SDXL styles.. "This is a SD XL checkpoint but the SD XL workload has not been installed."

### DISCUSSION THREAD

**Discussion by riccardobiagi** [Acly/krita-ai-diffusion#894] [Q&A]
Hi! I'm using a custom local server where I run my comfyui to connect to Krita. I'm on a macbook M1.

I have every file installed as the guide says and the plugin seems working ok with SD1.5. But if I want to use a SDXL style my SDXL models are not found. 

In Krita I go in Configure Image Diffusion > Styles, and in the model checkpoints dropdown list I see the warning "This is a SD XL checkpoint but the SD XL workload has not been installed." on every xl checkpoint that I have. 

In the client.log file I see at the end:
2024-06-30 18:15:42,936 INFO SD 1.5: supported
2024-06-30 18:15:42,937 INFO SD XL: missing 2 models
2024-06-30 18:15:42,937 INFO SD 3: missing 3 models

But I have no idea which models is talking about and what is "workload" to be installed.. 
Can someone please help me with that? I'm not good at this technical stuff.. 
Thank you

(I don't care abouto SD3).

**Comment by Acly** [comment]
Further up the client.log lists all models, and says whether they've been found or are missing.
